scripts/ddd/ddd

```diff
@@ -95,7 +95,11 @@ restore_mapping() {
 
 	# If we have stored USB_MAP, reload it
 	if [ -f ${DDD_MAP} ]; then
-		source ${DDD_MAP}
+        source ${DDD_MAP}
```

Comment:
I'm not entirely sure why we need to do this loop over everything instead of just getting DDD_MAP, is that to merge the old USB_MAP with the new one? I was planning on never needing this behavior (always load a fresh copy from disk before modifying) to prevent a race condition.